multidict/_multidict_c.c

```diff
@@ -0,0 +1,1587 @@
+#include "_multidict_c.h"
+#include "_pair_list.h"
+#include "_multidict_views.h"
+
+#include <structmember.h>
+
+// fix for VisualC complier used by Python 3.4
```

Comment:
Is this necessary if having in mind the current version of python 3.8?

Reply:
Python 3.4 is dropped but Python 3.5 is the minimal supported version now.
It will take a long when the minimal version became 3.8

Reply:
Dropped